cgra/test/CgraRTL_test.py

```diff
@@ -164,40 +172,43 @@ def init_param(topology, FuList = [MemUnitRTL, AdderRTL], data_bitwidth = 32):
   
   NocPktType = mk_multi_cgra_noc_pkt(ncols = num_terminals,
                                      nrows = 1,
+                                     ntiles = width * height,
                                      addr_nbits = addr_nbits,
                                      data_nbits = data_bitwidth,
-                                     predicate_nbits = 1)
+                                     predicate_nbits = 1,
+                                     ctrl_actions = num_commands,
+                                     ctrl_mem_size = ctrl_mem_size,
+                                     ctrl_operations = num_ctrl_operations,
+                                     ctrl_fu_inports = num_fu_inports,
+                                     ctrl_fu_outports = num_fu_outports,
+                                     ctrl_tile_inports = num_tile_inports,
+                                     ctrl_tile_outports = num_tile_outports)
+
   pick_register = [FuInType(x + 1) for x in range(num_fu_inports)]
   tile_in_code = [TileInType(max(4 - x, 0)) for x in range(num_routing_outports)]
   fu_out_code  = [FuOutType(x % 2) for x in range(num_routing_outports)]
   src_opt_per_tile = [[
-                # src dst vc_id opq cmd_type    addr operation predicate
-      CtrlPktType(0,  i,  0,    0,  CMD_CONFIG, 0,   OPT_INC,  b1(0),
-                  pick_register, tile_in_code, fu_out_code),
-                  # [TileInType(4), TileInType(3), TileInType(2), TileInType(1),
-                  #  # TODO: make below as TileInType(5) to double check.
-                  #  TileInType(0), TileInType(0), TileInType(0), TileInType(0)],
-  
-                  # [FuOutType(0), FuOutType(0), FuOutType(0), FuOutType(0),
-                  #  FuOutType(1), FuOutType(1), FuOutType(1), FuOutType(1)]),
-      CtrlPktType(0,  i,  0,    0,  CMD_CONFIG, 1,   OPT_INC, b1(0),
-                  pick_register, tile_in_code, fu_out_code),
+      CtrlPktType(0, 0,  i,  0,    0,  CMD_CONFIG, 0, OPT_INC, 0,
```

Comment:
The comments are accidentally removed? I was asking adding more to explain the `0, 0, 0, ...`.